horace_core/sqw/coord_transform/@ubmat_proj/get_line_proj.m

```diff
@@ -0,0 +1,12 @@
+function lp = get_line_proj(obj)
```

Comment:
If we can get a `line_proj`, why do we need to define `ubmat_proj` in the first place? Could it not just be a function?

Reply:
The code, which is in `ubmat_proj` was previously in `line_proj`. The code is responsible for processing legacy data and distribured in over more than one functions. Though it is theoretically possible to move the changes to set of functions, class is the best pattern for that as encapsulates and separates legacy data and legacy functions together. 

Now, if we do not want to maintain legacy data, we just drop `ubmat_proj`. In addition, it is possible and may be necessary to add specific code to this class. (was planning to do that but now see low priority for this kind of operations)